### PR TITLE
[Vectorize] Announcement for the anticipated blocking of legacy index creation

### PR DESCRIPTION
**Pull request by garvit-gupta**
### Summary

 Announcement for the anticipated blocking of legacy index creation.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx [hunk 1]
```diff
@@ -17,6 +17,8 @@ This document provides details around any transition steps that may be needed to
 
 Legacy Vectorize (V1) indexes are on a deprecation path. Support for these indexes would be limited and their usage is not recommended for any production workloads.
 
+Furthemore, the creation of legacy Vectorize indexes would be blocked by December 2024. Other operations would be unaffected and would remain functional.
```
Comment (netgusto) [inline src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx]: I'd replace would by will in there.

Comment (sejoker) [inline src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx]:
```suggestion
Furthemore, the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations would be unaffected and would remain functional.
```

File: src/content/partials/vectorize/vectorize-legacy.mdx [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 Please use the `wrangler vectorize --deprecated-v1` flag to create, get, list, delete and insert vectors into legacy Vectorize V1 indexes.
 
+Please note that the creation of legacy Vectorize indexes would be blocked by December 2024. Other operations would remain functional.
```
Comment (sejoker) [inline src/content/partials/vectorize/vectorize-legacy.mdx]:
```suggestion
Please note that the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will remain functional.
```

File: src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx [hunk 3]
```diff
@@ -17,6 +17,8 @@ This document provides details around any transition steps that may be needed to
 
 Legacy Vectorize (V1) indexes are on a deprecation path. Support for these indexes would be limited and their usage is not recommended for any production workloads.
 
+Furthemore, the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations would be unaffected and would remain functional.
```
Comment (sejoker) [inline src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx]:
```suggestion
Furthemore, the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will be unaffected and will remain functional.
```

**Review by sejoker — CHANGES_REQUESTED** [review]

**Review by sejoker — APPROVED** [review]

File: src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx (hunk 1 above)
Comment (garvit-gupta) [reply src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx]: @sejoker fixed this.

File: src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx [hunk 4]
```diff
@@ -17,6 +17,8 @@ This document provides details around any transition steps that may be needed to
 
 Legacy Vectorize (V1) indexes are on a deprecation path. Support for these indexes would be limited and their usage is not recommended for any production workloads.
 
+Furthemore, the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will be unaffected and will remain functional.
```
Comment (Oxyjun) [inline src/content/docs/vectorize/reference/transition-vectorize-legacy.mdx]:
```suggestion
Furthermore, you will no longer be able to create legacy Vectorize indexes by December 2024. Other operations will be unaffected and will remain functional.
```

File: src/content/partials/vectorize/vectorize-legacy.mdx [hunk 5]
```diff
@@ -6,6 +6,8 @@
 
 Please use the `wrangler vectorize --deprecated-v1` flag to create, get, list, delete and insert vectors into legacy Vectorize V1 indexes.
 
+Please note that the creation of legacy Vectorize indexes will be blocked by December 2024. Other operations will remain functional.
```
Comment (Oxyjun) [inline src/content/partials/vectorize/vectorize-legacy.mdx]:
```suggestion
Please note that by December 2024, you will not be able to create legacy Vectorize indexes. Other operations will remain functional.
```

**Review by Oxyjun — APPROVED** [review]

**Pull request merged**